### PR TITLE
Swifty result handling enhancement

### PR DESCRIPTION
**Pull request by jpalten**
Optional PR, in case you find it useful.

Added asAsset to CaptureContent
Extracted enums to separate file

### CODE REVIEW

File: CameraManager/CameraEnums.swift [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+public enum CameraOutputMode {
+    case stillImage, videoWithMic, videoOnly
+}
+
+public enum CameraOutputQuality: Int {
+    case low, medium, high
+}
+
+public enum CaptureResult {
```
Comment (bernikowich) [inline CameraManager/CameraEnums.swift]: Is it really needed in Swift 5?

File: CameraManager/CameraEnums.swift [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+            } else {
+                return nil
+            }
+        }
+    }
+
+    public var asData: Data? {
+
+        switch self {
+        case let .image(image): return image.jpegData(compressionQuality: 0.8)
```
Comment (bernikowich) [inline CameraManager/CameraEnums.swift]: 0.8? :)

**Comment by bernikowich** [comment]
Btw, your git config is broken.

**Comment by torrao** [comment]
Thanks for the input, but the idea behind Camera Manager is a single class that provides all the functionality, so extracting the enums to a separate file would mean that's no longer valid.

**Pull request closed without merging**